spec/requests/vacancies_spec.rb

```diff
@@ -1,5 +1,20 @@
 require "rails_helper"
 
 RSpec.describe "Vacancies", type: :request do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe "GET /vacancies" do
+    it "returns http success" do
+      get vacancies_path
+      expect(response).to have_http_status(:success)
+    end
+
+    it "redirects to root_path if current_user is not authenticated" do
```

Comment:
redirect to root_path if **user** is not authenticated

Reply:
@XaoGao ok